package.json

```diff
@@ -11,6 +11,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "aws-sdk": "^2.910.0",
```

Comment:
AWS environment provide by default the AWS-SDK, this package is not necessary in the dependencies sections only in the dev-dependencies to perform tests in local.